x-pack/plugins/security_solution/public/common/utils/default_date_settings.ts

```diff
@@ -84,9 +84,11 @@ export const getIntervalSettings = (uiSettings = true): Policy => {
 export const parseDateWithDefault = (
   dateString: string,
   defaultDate: moment.Moment,
-  roundUp: boolean = false
+  roundUp: boolean = false,
+  momentInstance?: typeof moment,
+  forceNow?: Date
```

Comment:
these new parameters are missing in JSDoc

Reply:
added